package.json

```
@@ -44,7 +44,7 @@
   "productName": "Sheikah-Witnet-wallet",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/witnet/sheikah.git"
+    "url": "git+https://github.com/gabaldon/sheikah.git"
```

Could we discard the changes in this file?
```suggestion
    "url": "git+https://github.com/witnet/sheikah.git"
```